5.5/test/run

```diff
@@ -108,15 +110,15 @@ function run_change_password_test() {
   create_container "testpass1" -e MYSQL_USER=user -e MYSQL_PASSWORD=foo \
     -e MYSQL_DATABASE=db -v ${tmpdir}:/var/lib/mysql/data:Z
   test_connection testpass1 user foo
-  docker stop $(get_cid testpass1)
+  docker stop $(get_cid testpass1) >/dev/null
 
   # Create second container with changed password
   create_container "testpass2" -e MYSQL_USER=user -e MYSQL_PASSWORD=bar \
     -e MYSQL_DATABASE=db -v ${tmpdir}:/var/lib/mysql/data:Z
   test_connection testpass2 user bar
 
   # The old password should not work anymore
-  if mysql_cmd "$(get_container_ip testpass2)" user foo -e 'SELECT 1;'; then
+  if mysql_cmd "$(get_container_ip testpass2)" user foo -e 'SELECT 1;' 2>/dev/null; then
```

Comment:
To make output clean. For example, it was like this before:
```
ERROR 2003 (HY000): Can't connect to MySQL server on '172.17.0.7' (111)
ERROR 2003 (HY000): Can't connect to MySQL server on '172.17.0.7' (111)
172.17.0.8 successfully registered as SLAVE for 172.17.0.7
```
And now there is no such errors that actually isn't errors.

Do you think that we shouldn't hide them?